src/core/src/UsdUtil.cpp

```diff
@@ -296,10 +297,12 @@ pxr::CesiumGeoreference defineCesiumGeoreference(const pxr::SdfPath& path) {
 
 pxr::CesiumTilesetAPI defineCesiumTileset(const pxr::SdfPath& path) {
     auto stage = getUsdStage();
-    auto xform = pxr::UsdGeomXform::Define(stage, path);
-    assert(xform.GetPrim().IsValid());
+    auto mesh = pxr::UsdGeomMesh::Define(stage, path);
```

Comment:
It's unfortunate that `UsdGeomBoundable` doesn't work. It might be worth digging into the Omniverse code some more or reaching out to understand why it needs to be a mesh.

Reply:
I think it comes down to there are only certain types of prims that can created in the stage. There's no way to create a `UsdGeomBoundable` directly. Its constructor needs an existing prim or a schema. In other words, it doesn't have a `Define` method. I was wondering if an `extents` property could be added to `CesiumTilesetAPI`. We could try, but it does seem like the Viewport looks for a mesh.

Reply:
> unless we created our own concrete type that inherits from `UsdGeomBoundable`

That actually sounds kind of appealing since the `UsdGeomMesh` attributes aren't relevant to tilesets and clutter the UI. It could also lead to confusion.

If we had our own concrete type we could keep it minimal. Also I think it makes sense for CesiumTileset to be a concrete type like the other Cesium types.

Reply:
Yeah, I was going to ask why not a fan of our own concrete type in standup. Aside from the minimalness @lilleyse mentions, we could add in future things without the potential breaking-change of a new Type in the USD file.

Reply:
@lilleyse, @corybarr - Let's definitely talk about this in stand-up today. I do see the merit in making CesiumTileset a concrete type that inherits from `UsdGeomBoundable`, but I also have some concerns about it causing some unintended side-effects. Rather than elaborate here, I'd rather discuss in person.